web/src/dropdown_widget.ts

```diff
@@ -398,7 +398,7 @@ export class DropdownWidget {
                 });
 
                 // Click on item.
-                $popper.one("click", ".list-item", (event) => {
+                $popper.on("click", ".list-item", (event) => {
```

Comment:
What's the reason for this change?

Reply:
The saved replies can be deleted by clicking on the trash icon in dropdown. But with `popper.one` only one click is allowed in the dropdown. So I changed it to `popper.on` to allow multiple deletions without dropdown closing.